great_expectations/data_context/data_context/file_data_context.py

```diff
@@ -44,6 +47,14 @@ def __init__(
     def _init_variables(self) -> FileDataContextVariables:
         raise NotImplementedError
 
+    def _save_project_config(self) -> None:
```

Comment:
Is it worth renaming this method to `_save_project_config_to_disk` for clarity? This would apply to all data contexts.

Reply:
update: after discussion, this is being renamed back to `_save_project_config()`.